Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -421,25 +421,44 @@ public static bool CalculateSuccessfulHit(Entity.DaggerfallEntity attacker, Enti
             Entity.PlayerEntity player = GameManager.Instance.PlayerEntity;
             Entity.EnemyEntity AITarget = null;
 
-            // Apply random modifier
-            int[] randomMods = { 0, 1, 1, 1, 1, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 6 };
-            int randomMod = randomMods[(UnityEngine.Random.Range(0, 19 + 1))];
+            int armorValue = 0;
 
-            chanceToHit += randomMod;
+            if (target == player)
+            {
+                // Choose struck body part
+                int[] bodyParts = { 0, 0, 1, 1, 1, 2, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 6 };
+                int struckBodyPart = bodyParts[(UnityEngine.Random.Range(0, 19 + 1))];
```

Comment:
This is perfectly OK for now, but consider using bodyParts.Length instead of 19+1 for upper value of Range(). This ensures the correct range will still be used if the array ever changes size.